### PR TITLE
Create docker-compose to redis-cluster and fix tests

### PR DESCRIPTION
**Pull request by arthur29**
WHY
======

Currently, Podium only has the environment to test Redis standalone. This PR proposes to create a docker-compose file that will create a proper environment to test these new features.

WHAT WAS DONE
======
* Create `docker-compose.yaml`
* Fix tests

### CODE REVIEW

File: build/package/initialize_cluster.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+HOSTS=$(echo $CLUSTER_HOSTS | tr " " "\n")
+i=0
+HOST_PORTS=()
+for element in $HOSTS; do
+  HOST_NAME=$(echo $element | cut -d ":" -f 1)
+  PORT=$(echo $element | cut -d ":" -f 2)
+  IP=$(host ${HOST_NAME} | grep "has address" | cut -d " " -f 4)
+
+  HOST_PORTS=("${HOST_PORTS[@]}" "$(echo $IP:$PORT)")
+  i=$(($i + 1))
```
Comment (mathnogueira) [inline build/package/initialize_cluster.sh:12]: It seems you are not using this variable at all in this script.

**Review by leonardo2204 — APPROVED** [review]
Nice refactoring, btw :D

File: deployments/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    build:
+      context: ../
+      dockerfile: ./build/RedisClusterDockerfile
+    image: redis-cluster:6.2.1-alpine
+    networks:
+      podium:
+        aliases:
+          - redis-node-2
+
+  initialize-cluster:
```
Comment (mathnogueira) [inline deployments/docker-compose.yaml:31]: Why don't you use a noun here? Docker containers usually are named after nouns. For example: `redis-cluster`

Comment (arthur29) [reply deployments/docker-compose.yaml:31]: How this container is just a sidecar I prefer a name, as an action to be executed. Why do you think?

**Review by mathnogueira — APPROVED** [review]

**Pull request merged**